lib/utils/image.dart

```diff
@@ -137,11 +137,11 @@ Size getWEBPImageDimensions(Uint8List bytes) {
 void uploadImage(BuildContext context, ImageBloc imageBloc, {bool postImage = false, String? imagePath}) async {
   final ImagePicker picker = ImagePicker();
   String path;
-  if (imagePath?.isEmpty ?? false) {
+  if (imagePath == null || imagePath.isEmpty) {
```

Comment:
Yeah, I chose to explicitly write the conditions in this case because I think that's what initially caused the confusion which led up to the issue (not fully interpreting/missed during code review what `imagePath?.isEmpty ?? false` meant)